code/modules/lighting/lighting_atom.dm

```diff
@@ -48,16 +48,6 @@
 /atom/proc/extinguish_light(force = FALSE)
 	return
 
-// If we have opacity, make sure to tell (potentially) affected light sources.
-/atom/movable/Destroy()
-	var/turf/T = loc
-	. = ..()
-	if(opacity && istype(T))
-		var/old_has_opaque_atom = T.has_opaque_atom
-		T.recalc_atom_opacity()
-		if(old_has_opaque_atom != T.has_opaque_atom)
-			T.reconsider_lights()
-
```

Comment:
this makes me nervous lol

Reply:
This is all added back in `/atom/movable/Destroy()` in `atoms_movable.dm`

Reply:
I figured, I just don't know if the redefinition borked something long ago and it's now being revived

Reply:
It doesn't because this redefine calls parent, which due to some incredibly magic Byond stuff, calls the first `/atom/movable/Destroy()`